src/wasm-lib/kcl/src/ast/types.rs

```diff
@@ -3195,6 +3195,18 @@ pub fn parse_json_value_as_string(j: &serde_json::Value) -> Option<String> {
     }
 }
 
+/// Convert JSON value to bool.  Everything is true except for null and false.
+pub fn json_to_bool(j: &serde_json::Value) -> bool {
```

Comment:
I would rather this fail if used with anything other than bool. We can always relax it later. But I think the JS and Python style "truthiness" stuff will be confusing.